zeek/README.md

```diff
@@ -0,0 +1,201 @@
+## Overview
+
+[Zeek][6] is the world's leading platform for network security monitoring. It interprets what it sees and creates compact, high-fidelity transaction logs, and file content. It can create fully customized output, suitable for manual review on disk or in a more analyst-friendly tool like a security and information event management (SIEM) system.
+
+This integration ingests Connection logs, DNS and DHCP logs, Network Protocols, Files, Detections, and Miscellaneous event types with the integration log pipeline to enrich the logs and normalizes data to datadog standard attributes. This integration offers visualisations to gain detailed insights into network connections, DNS and DHCP activity, detailed network protocol analysis, file analysis and certificates, security detection and observation, compliance monitoring, etc. via dashboards.
+
+## Setup
+
+### Installation
+
+To install the Zeek integration, use the `datadog-agent integration install` command and follow the steps below. For more information, see the [Integration Management][7] documentation.
+
+**Note**: This step is not necessary for Agent version >= 7.52.0.
+
+Linux command
+  ```shell
+  sudo -u dd-agent -- datadog-agent integration install datadog-zeek==1.0.0
+  ```
+
+#### Opensource Zeek
+1. [Install the Agent][4] on your Zeek machine.
+2. Setup json-streaming-logs package
+   - Install [Corelight Zeek plugin][5] for JSON logging
+     ```
+     /opt/zeek/bin/zkg install corelight/json-streaming-logs
+     ```
+   - Load ZKG packages
+     ```
+     echo -e "\n# Load ZKG packages\n@load packages" >> /opt/zeek/share/zeek/site/local.zeek
+     ```
+   - Restart the zeek
+     ```
+     /opt/zeek/bin/zeekctl install
+     ```
+     ```
+     /opt/zeek/bin/zeekctl restart
+     ```
+
+#### Corelight Zeek
+* Have the [Datadog Agent][4] installed and running.
+
+### Configuration
+
+#### Log collection
+
```

Comment:
Will this be filled out?

Reply:
No explicit need to add configuration since it gets covered in below sections.

Reply:
Can these subheadings be removed then?

Reply:
Done